docs/modules/operation/pages/flows/basic.adoc

```diff
@@ -0,0 +1,127 @@
+
+[[flows-basic]]
+= Start here: basic flows setup
+
+This section describes how to get started with flows in a basic setup to collect, enrich (classify), persist, and visualize flows.
+
+== Requirements
+
+Make sure you have the following before you set up flows:
+
+* OpenNMS up and running.
+* Device(s) that sends flows visible to OpenNMS and monitored with SNMP.
+* Elasticsearch cluster set up with the link:https://github.com/OpenNMS/elasticsearch-drift-plugin[Elasticsearch Drift plugin] installed on every Elasticsearch node.
+** The Drift plugin persists and queries flows that {page-component-title} collects.
+The Drift version must match the targeted Elasticsearch version.
+** (optional) Configure Elasticsearch variables like `search.max_buckets` if the default values are not sufficient.
+** (optional) Create a job to clean the jobs so that the disk does not fill up; for example, keep X days of flows.
+Filled disks are a challenging problem to address for non-Elasticsearch experts.
+We recommend the Elasticsearch https://www.elastic.co/guide/en/elasticsearch/client/curator/current/index.html[Curator tool] to do this.
+** Monitor the Elasticsearch stack in OpenNMS to get an alarm if it goes down.
+* Set up xref:helm/ROOT/index.adoc#welcome-index[OpenNMS Helm] with Grafana (to visualize flows)
+** Configure the xref:getting_started/basic_walkthrough.adoc#bw-ds-setup[flow and performance data sources]
+
+== Configure OpenNMS to communicate with Elasticsearch
+
+OpenNMS needs to talk to Elasticsearch and know where to store the flows data it collects (persistence).
+
+From a Karaf shell on your {page-component-title} instance, update `$OPENNMS_HOME/etc/org.opennms.features.flows.persistence.elastic.cfg` to configure the flow persistence to use your Elasticsearch cluster:
+
+[source, console]
+----
+$ ssh -p 8101 admin@localhost
+...
+admin@opennms()> config:edit org.opennms.features.flows.persistence.elastic
+admin@opennms()> config:property-set elasticUrl http://elastic:9200
+admin@opennms()> config:update
+----
+
+We also recommend setting the following:
+
+[source, xml]
+----
+# ElasticSearch persistence configuration
+elasticUrl = http://10.10.3.218:9200
+connTimeout = 30000
+readTimeout = 300000
+settings.index.number_of_replicas = 0
+settings.index.number_of_shards=1
+settings.index.refresh_interval=10s
+elasticIndexStrategy=daily
+----
+
+See <<elasticsearch/introduction.adoc#ga-elasticsearch-integration-configuration, General Elasticsearch Configuration>> for a complete set of options.
```

Comment:
```suggestion
NOTE: See <<elasticsearch/introduction.adoc#ga-elasticsearch-integration-configuration, General Elasticsearch Configuration>> for a complete set of options.
```

Reply:
NH style guide prefers to limit the use of notes, and since this is a separate paragraph with a link, I think it already stands out.